features/schemas/ua_status.json

```
@@ -0,0 +1,246 @@
+{
+  "type": "object",
+  "properties": {
+    "_doc": {
+      "type": "string"
+    },
+    "_schema_version": {
+      "type": "string"
```

I discovered "const" recently so we can do this for the schema version
```suggestion
      "type": "string",
      "const": "0.1"
```